manuscript/01_introduction.md

```diff
@@ -58,14 +58,14 @@ How do we collect a reasonable subset of data and quantify the
 uncertainty in the process to produce a good guess at who will win?
 
 
-### Motivating example, predicting the weather
+### Motivating example: predicting the weather
 
 When a weatherman tells you the probability that it will rain tomorrow is
 70%, they're trying to use historical data
-to predict tomorrow's weather - and to actually attach a probability to it.
+to predict tomorrow's weather – and to actually attach a probability to it.
 That probability refers to population.
 
-#### Motivating example, brain activation
+### Motivating example: brain activation
```

Comment:
Pulls this section up to be a sibling of the other examples instead of a sub-section of previous "predicting the weather" example.